src/main/scala/org/clulab/coref/Coref.scala

```diff
@@ -347,11 +347,9 @@ class Coref {
 
       if (mentions.isEmpty) Nil
       else {
-        if (debug) {
-          println("BEFORE COREF")
-          displayMentions(mentions, mentions.head.document)
-          println("Starting coref...")
-        }
+        logger.debug("BEFORE COREF")
+        displayMentions(mentions, mentions.head.document)
```

Comment:
The method that returns a string is `display.summarizeMentions` (same signature).  Maybe this could be changed to something like `logger.debug(display.summarizeMentions(mentions, mentions.head.document))`?